packages/mock-repo/package.json

```diff
@@ -24,6 +24,7 @@
   "dependencies": {
     "@ui-autotools/registry": "^1.0.7",
     "@ui-autotools/scripts": "^1.0.19",
+    "@ui-autotools/utils": "^1.0.6",
```

Comment:
Which of utils does it use?

Reply:
consoleLog

Reply:
Don't see it anywhere.

Reply:
Whoops, my bad. Thought it was in Eyes. Yeah, deleted this dep